### PR TITLE
Create get_installed_themes.lua

### PR DESCRIPTION
**Pull request by Subarctic2796**
Hi
(I am new to git, so please bare with me)

I made a new file as I didn't know where to put this but I am happy to change it.

Basically I am returning a lua table that has 2 properties.
1) all_installed_themes: this gets all of the currently installed themes from vim itself so you don't have to manually enter the name of all the themes.
2) default_vim_themes: this is a list of all of the default themes installed in neovim when you first download nvim

I think it is good to have this as you can choose to filter out the default installed themes later.

I made a simple theme picker using telescope that I would be happy to share with you if you want to see it

### CODE REVIEW

**Comment by Subarctic2796** [comment]
Closed as it seems like there is no interest in this. Don't mind I am no longer working on this anyways

**Comment by zaldih** [comment]
First of all, I apologize profusely. I thought I had responded to you a long time ago, and it ended up being forgotten. I always appreciate any kind of contribution and try to foster a good environment, especially with new users who are taking their first steps. Again, sorry for this.

The reason I didn't accept this PR was because I had planned to make some changes to the project in the future where it would be possible to choose between the user's themes and the system's themes separately. Of course, this hasn't happened yet due to a lack of time.

To summarize, there was nothing wrong with the PR; I was just waiting for other changes to be made first so that the integration would make sense.

I wish you all the best in contributing to open source!

**Comment by Subarctic2796** [comment]
All good. I honestly forgot about it myself I was just clearing out my repos.
